lib/commons/dom/visibility-functions.js

```diff
@@ -0,0 +1,171 @@
+import {
```

Comment:
We don't make these functions public on `axe` so can't really test them. We could either make them all available like we do with the text methods we mimicked or add them to the `_thisWillBeDeletedDoNotUse` object. 

Since this is just a refactor to make the code more readable, I'm sort of OK if this file isn't tested and we have all the tests in the visibility functions.

Reply:
Disagree. I'm uncomfortable when we do it just for pulling a method into its own file. With whole new methods I think we need to make the effort. These are easy tests to write. Let's just get them done.